index.html

```diff
@@ -36,5 +36,137 @@ <h3 class="intro-2">LET'S CONNECT</h3>
+                <div class="back-end">Back End Dev</div>
+                <div class="dot"></div>
+                <div class="year">2015</div>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+            </ul>
+            <button class="project-button">See project</button>
+        </div>
+        <div class="work">
+            <img src="images/Snapshoot Portfolio (1).png" alt="tonic">
+            <h2 class="multi">Multi-Post Stories</h2>
+            <div class="works">
+                <div class="canopy">CANOPY</div>
+                <div class="dot"></div>
+                <div class="back-end">Back End Dev</div>
+                <div class="dot"></div>
+                <div class="year">2015</div>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+            </ul>
+            <button class="project-button">See project</button>
+        </div>
+        <div class="work">
+            <img src="images/Snapshoot Portfolio (2).png" alt="tonic">
+            <h2 class="tonic">Tonic</h2>
+            <div class="works">
+                <div class="canopy">CANOPY</div>
+                <div class="dot"></div>
+                <div class="back-end">Back End Dev</div>
+                <div class="dot"></div>
+                <div class="year">2015</div>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+            </ul>
+            <button class="project-button">See project</button>
+        </div>
+        <div class="work">
+            <img src="images/Snapshoot Portfolio (3).png" alt="tonic">
+            <h2 class="multi">Multi-Post Stories</h2>
+            <div class="works">
+                <div class="canopy">CANOPY</div>
+                <div class="dot"></div>
+                <div class="back-end">Back End Dev</div>
+                <div class="dot"></div>
+                <div class="year">2015</div>
+            </div>
+            <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="skills">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+            </ul>
+            <button class="project-button">See project</button>
+        </div>
+    </section>
+    <section class="about-me">
+        <div>
```

Comment:
I really like your styling so far but check this out in the figma: 
![image](https://user-images.githubusercontent.com/107795045/227628976-e050ce22-1c48-4019-9d00-b5f6914f974e.png)
There is a ```border-top-radius``` on this section which you may have forgotten to add so please do add it to make your project look exactly like the template.

Your design: 
![image](https://user-images.githubusercontent.com/107795045/227629703-26f0d2f4-fe30-41c4-afce-ff17510341d5.png)